packages/mobile/src/utils/useClipboard.ts

```diff
@@ -1,17 +1,25 @@
+import Clipboard from '@react-native-community/clipboard'
 import { useEffect, useState } from 'react'
-import { AppState, Clipboard } from 'react-native'
+import { AppState, Platform } from 'react-native'
 import Logger from 'src/utils/Logger'
 
 const CLIPBOARD_CHECK_INTERVAL = 1000 // 1sec
 
-export function useClipboard(): string {
+export function useClipboard(): [boolean, string] {
```

Comment:
Agreed, how about instead of a string we return a function that returns the contents of the clipboard? The API would still look weird (why a function instead of a string directly?) but it would reduce the error-proneness imo.

Reply:
I took another pass as this, but I'm not convinced by the result. It still looks weird to me. Maybe different naming can help? I added a comment to try to clarify a bit..

Reply:
Ok I've thought about something. I'll detail it tomorrow. 
In the mean time, I think it's fine to still merge this as is.